internal/provider/resource_consul_cluster.go

```diff
@@ -662,6 +655,11 @@ func resourceConsulClusterUpdate(ctx context.Context, d *schema.ResourceData, me
 		version := d.Get("min_consul_version")
 		newConsulVersion := input.NormalizeVersion(version.(string))
 
+		// Attempt to get the latest patch version of the given min_consul_version.
+		if patch, found := consul.GetLatestPatch(newConsulVersion, upgradeVersions); found {
```

Comment:
Should we error out if not found?

Reply:
I don't think we need to as we immediately have version checking logic [here](https://github.com/hashicorp/terraform-provider-hcp/blob/08ad21a4f63bf4ca12ec5786067519f8fdac902a/internal/provider/resource_consul_cluster.go#L294).